accepted/2024/PMUI-Readme-rendering.md

```diff
@@ -22,15 +22,29 @@ The PM UI will be updated to have tabs for the Package Details and the the READM
 The README will be available in all the PM UI tabs.
 It will also be displayed for both the solution level and project level package managers.
 
+The selected tab will be maintained between sessions and when switching packages. 
```

Comment:
Is this implemented already or is a PR coming?

Reply:
Does any of this consider whether the source supports readmes?

Reply:
> Is this implemented already or is a PR coming?

Already implemented, just clarifying that even if the selected package does not have a README we will stay on the tab.

Reply:
> Does any of this consider whether the source supports readmes?

The start of the section says "This tab is rendered if a remote source available to download the readme or the Installed tab is selected and the selected version is found in the global packages folder." I feel like that covered it.